.github/workflows/ci.yaml

```diff
@@ -0,0 +1,108 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  devcontainer:
+    # Deduplicate jobs from pull requests and branch pushes within the same repo.
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository
+    uses: ./.github/workflows/_devcontainer.yaml
+
+  bundler_lint:
+    # Deduplicate jobs from pull requests and branch pushes within the same repo.
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository
+    uses: ./.github/workflows/_bundler_lint.yaml
+
+  bundler_test:
+    # Deduplicate jobs from pull requests and branch pushes within the same repo.
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository
+    uses: ./.github/workflows/_bundler_test.yaml
+
+  bundler_container:
+    # Deduplicate jobs from pull requests and branch pushes within the same repo.
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
This should already have been checked by the `needs` jobs.

Reply:
It is, I just duplicated it to be explicit. Do you think it'd be better off without?

Reply:
Yeah maybe it's fine to be explicit, if it is just `needs: check` it might be nicer. I wonder if it that will produce a lot of extra connections in the workflow, though?